src/components/StoriesGroup/README.md

```diff
@@ -4,15 +4,16 @@ Component for displaying group of stories. It looks like a carousel in a modal w
 
 ### PropTypes
 
-| Property            | Type                                      | Required | Default | Description                                                |
-| :------------------ | :---------------------------------------- | :------- | :------ | :--------------------------------------------------------- |
-| open                | `Boolean`                                 | ✓        |         | Visibility flag                                            |
-| groups              | `StoriesGroupItem[]`                      | ✓        |         | List of groups of stories to display                       |
-| initialStoryIndex   | `[groupIndex: Number, itemIndex: Number]` |          | [0, 0]  | Index of the first story to be displayed                   |
-| onClose             | `Function`                                |          |         | Action on close                                            |
-| onItemSelect        | `Function`                                |          |         | Action when switching to story                             |
-| disableOutsideClick | `Boolean`                                 |          | true    | If `true`, do not close stories on click outside           |
-| maxSliderItemsCount | `Number`                                  |          | 12      | Positive maximum number of slider thumbnails on the screen |
+| Property            | Type                                      | Required | Default                       | Description                                                   |
+| :------------------ | :---------------------------------------- | :------- | :---------------------------- | :------------------------------------------------------------ |
+| open                | `Boolean`                                 | ✓        |                               | Visibility flag                                               |
+| groups              | `StoriesGroupItem[]`                      | ✓        |                               | List of groups of stories to display                          |
+| initialStoryIndex   | `[groupIndex: Number, itemIndex: Number]` |          | [0, 0]                        | [DEPRICATED] Initial index of the first story to be displayed |
+| index               | `StoriesGroupIndex`                       |          | {groupIndex: 0, itemIndex: 0} | Index of the current story to be displayed                    |
```

Comment:
I think it is better to use as array like `initialStoryIndex`, untill next major, but I'm not sure

Reply:
There's no point to use an outdated interface for a new field, and then rewrite it anyway and force the user also rewrite this place.